www/www-deploy.yml

```diff
@@ -69,8 +69,6 @@
       dest: /security/advisories/
     - match: "~/info/vulnerabilities/(?<link>.*[^/])/?$"
       dest: /security/advisories/$link/
-    - match: "~/info/(?<link>.*)$"
```

Comment:
What now handles the `/info/` URLs?

Reply:
They'll just go the front page of the new website, won't they? I don't know what they are to redirect them individually

Reply:
Or actually to our new 404 page I guess. I could just redirect them to the home page if you prefer?

Reply:
Unless, I guess there must be a list somewhere because the legacy/info/ URLs are all redirects anyway - any idea where they live @sbesson ?

Reply:
I do not think dropping them is an option. A [search](https://github.com/search?q=%22openmicroscopy.org%2Finfo%22&type=Code&utf8=%E2%9C%93) on GitHub only shows minimally 1K usages and some of those `info` links are used for the security vulnerability pages or attribution links in the core xsd schema files.

The list is defined in the Apache configuration file for `www-legacy`. Reducing the `www/info -> www-legacy/info -> link` redirection path might be valuable. Happy to have a call about this tomorrow but I would propose we handle it outside the scope of this PR.

Reply:
The security ones are handled separately above anyway but yes, we can discuss tomorrow